jobs/curator/templates/bin/purge_logs

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
If this were renamed to just `run`, I think this job could have additional value as being errand-friendly which may work better in some users' BOSH lifecycles.